oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHWDiskStore.java

```diff
@@ -66,6 +67,15 @@
     private static final String ID_MODEL = "ID_MODEL";
     private static final String ID_SERIAL_SHORT = "ID_SERIAL_SHORT";
 
+    private static final String DM_UUID = "DM_UUID";
+    private static final String DM_VG_NAME = "DM_VG_NAME";
+    private static final String DM_LV_NAME = "DM_LV_NAME";
+    private static final String LOGICAL_VOLUME_GROUP = "Logical Volume Group";
+    private static final String DEV_NAME = "DEV_NAME";
+    private static final String DEV_PATH = "DEV_PATH";
+    private static final String DEV_LOCATION = "/dev";
```

Comment:
I think you'll find setting this constant to include the trailing slash will save you time elsewhere. :)

Reply:
yes makes it much easier.